db/migrate/20190606114651_remove_organization_id_column_from_user.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOrganizationIdColumnFromUser < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :users, :organization_id, :integer
```

Comment:
why do u need 2 separate migration to perform 1 action?
removing the column automatically removes all related indexes